shielddash/studies/views.py

```diff
@@ -16,3 +20,44 @@ class StudyListView(ListAPIView):
     filter_fields = ('start_time', 'end_time')
     filter_class = StudyFilter
     renderer_classes = (StudyJSONRenderer,)
+
+
+@api_view(['GET'])
+def study_detail(request, study_id):
+    study = get_object_or_404(Study, pk=study_id)
+    cursor = connection.cursor()
+    sql = """
+        SELECT
+            channel,
+            variation,
+            SUM(completed::int) AS completed,
+            SUM(ineligible::int) AS ineligible,
+            SUM(installed::int) AS installed,
+            SUM(left_study::int) AS left_study,
+            SUM(seen1::int) AS seen1,
+            SUM(seen2::int) AS seen2,
+            SUM(seen3::int) AS seen3,
+            SUM(seen7::int) AS seen7
+        FROM studies_state
+        WHERE study_id=%s
+        GROUP BY channel, variation
+        ORDER BY channel, variation
+    """
```

Comment:
I didn't even try. I wasn't sure if the ORM would be able to do the casting correctly and it's really not that hard of a SQL query to grok. Using the ORM would probably help to avoid the custom SQL code below. I could attempt it if you think it's worth it but I don't mind how this looks.